model/src/main/scala/hmda/model/fi/Filing.scala

```diff
@@ -0,0 +1,9 @@
+package hmda.model.fi
+
+sealed trait FilingStatus
+case object NotStarted extends FilingStatus
+case object InProgress extends FilingStatus
+case object Completed extends FilingStatus
+case object Cancelled extends FilingStatus
+
+case class Filing(id: String = "", fid: String = "", status: FilingStatus = NotStarted)
```

Comment:
Agreed.  I'd even suggest taking it a step further, and instead of a String, have a specific type for the filing period, so it has to be explicit everywhere.